keyboards/earth_rover/readme.md

```diff
@@ -0,0 +1,22 @@
+# earth_rover
+
+![earth_rover](https://pbs.twimg.com/media/Eqh29nYUUAA-9NE?format=jpg)
+
+*A short description of the keyboard/project*
```

Comment:
```suggestion
A prism-shaped keypad assembled from MX-compatible single-key PCBs.
```

Reply:
I think this description is for another keyboard. I update this line with another words.